azure/v15.0.0/release.yaml

```diff
@@ -1,10 +1,10 @@
 # Generated with:
-# devctl release create --name 15.0.0-beta1 --provider azure --base 14.1.5 --app chart-operator@2.13.1 --app cluster-autoscaler@1.20.2 --component app-operator@4.4.0 --component azure-operator@5.6.1-k8s-1-20 --component cert-operator@1.0.1 --component cluster-operator@0.27.1 --component kubernetes@1.20.6
+# devctl release create --name 15.0.0 --provider azure --base 14.1.5 --app chart-operator@2.13.1 --app cluster-autoscaler@1.20.2 --component app-operator@4.4.0 --component azure-operator@5.6.1-k8s-1-20 --component cert-operator@1.0.1 --component cluster-operator@0.27.1 --component kubernetes@1.20.6
```

Comment:
Dude, this will be a beta release

Reply:
I am trying to figure out why validation is failing, we can revert to beta1 later.